### PR TITLE
Add plugin for back office asset

### PR DESCRIPTION
**Pull request by ar4s**
Changes:
 - add ralph3_back_office_asset
 - add ralph3_data_center_asset
 - mark ralph3_asset as a deprecated plugin

TODO:
- [x] refactor update_asset
- [x] add test for new plugin

### CODE REVIEW

File: src/ralph_scrooge/plugins/collect/ralph3_asset.py [hunk 1]
```diff
@@ -338,17 +381,42 @@ def ralph3_asset(**kwargs):
...
+        'ralph3_service_environment',
+        'ralph3_asset_model'  # TODO: back office asset model
+    ]
+)
+def ralph3_back_office_asset(**kwargs):
+    """
+    Updates back office assets and usages
+    """
+    date = kwargs['today']
+    depreciation_usage = get_depreciation_usage(date=date)
```
Comment (mkurek) [inline src/ralph_scrooge/plugins/collect/ralph3_asset.py]: I'm not sure if using the same depreciation usage type for DC assets and BO assets is good idea. I'd consider separating it into two independent usage types.

File: src/ralph_scrooge/plugins/collect/ralph3_asset.py [hunk 2]
```diff
@@ -173,9 +173,67 @@ def update_usage(daily_asset_info, warehouse, usage_type, value, date):
...
+        except ServiceEnvironment.DoesNotExist:
+            service_environment = unknown_service_env
+            logger.warning(
+                'Invalid service environment for BO Asset {}: {} - {}'.format(
+                    bo_asset_repr, data['service_env']['service_uid'],
+                    data['service_env']['environment']
+                )
+            )
+    warehouse = Warehouse.objects.get(pk=1)
+    asset_info, asset_info_created = get_asset_info(
```
Comment (mkurek) [inline src/ralph_scrooge/plugins/collect/ralph3_asset.py:215]: consider creating another subclass of `PricingObject`, like `BackOfficeAssetInfo` for BO assets - it's the same situation as with depreciation.

File: src/ralph_scrooge/models/back_office.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+            defaults=dict(
+                pricing_object=self,
+                asset_info=self,
+                service_environment=self.service_environment,
+            )
+        )[0]
+
+
+class DailyBackOfficeAssetInfo(DailyPricingObject):
+    asset_info = db.ForeignKey(
```
Comment (mkurek) [inline src/ralph_scrooge/models/back_office.py]: i'm not sure if it should be named `asset_info` or `back_office_asset_info`

File: src/ralph_scrooge/models/back_office.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        BackOfficeAssetInfo,
+        related_name="daily_info_set",
```
Comment (mkurek) [inline src/ralph_scrooge/models/back_office.py]: `daily_back_office_asset_info_set`? :)

File: src/ralph_scrooge/models/back_office.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
```
Comment (mkurek) [inline src/ralph_scrooge/models/back_office.py]: why did you created it in separated file, not as a part of `pricing_object.py`?

File: src/ralph_scrooge/plugins/collect/ralph3_asset.py [hunk 6]
```diff
@@ -384,15 +460,53 @@ def ralph3_asset(**kwargs):
             type='BU',
         ),
     }
+    return _update_asset_and_usages(
+        date=date, usages=usages, ralph_endpoint='data-center-assets',
+        update_asset_func=update_data_center_asset
+    )
+
+
+@plugin_runner.register(
```
Comment (mkurek) [inline src/ralph_scrooge/plugins/collect/ralph3_asset.py:532]: by convention, registration of plugins should be at the end of the file

**Review by mkurek — APPROVED** [review]

**Review by MarekBleschke — APPROVED** [review]

**Pull request merged**